packages/nitro-rpc-client/scripts/client-runner.ts

```diff
@@ -139,6 +139,7 @@ yargs(hideBin(process.argv))
           waitForRPCServer(4005, yargs.waitduration),
           waitForRPCServer(4006, yargs.waitduration),
           waitForRPCServer(4007, yargs.waitduration),
+          waitForRPCServer(4008, yargs.waitduration),
```

Comment:
Nit: could make this more robust to future refactors by pushing the ports we used into an array that is read from here?